### PR TITLE
move helpers declaration into a separate global variable

### PR DESCRIPTION
**Pull request by tyv**
Now all helpers are prepended as global variable to the top of resulted file.

### CODE REVIEW

File: techs/js-babel.js [hunk 1]
```diff
@@ -34,14 +39,14 @@ module.exports = require('enb/lib/build-flow').create()
...
-                        transform.result.code, 
+                    concat.add(sourcemap.normalizeFileName(transform.file.fullname),
+                        transform.result.code,
                         JSON.parse(node.map.toString()));
                 }
             })
 
             fs.writeFileSync(target + '.map', concat.sourceMap);
-            return concat.content;
+            return babel.buildExternalHelpers() + '\n' + concat.content;
```
Comment (s-panferov) [inline techs/js-babel.js]: You can't concat this way because it breaks source maps. Please use `concat.add`

File: techs/js-babel.js [hunk 2]
```diff
@@ -4,16 +4,21 @@ var path = require('path'),
...
     .name('js-babel')
     .target('target', '?.browser.js')
     .defineOption('babelOptions')
-    .useFileList(['vanilla.js', 'js', 'browser.js'])
+    .useFileList(['vanilla.js', 'js', 'browser.js', 'jsx'])
     .builder(function (files) {
-        var babelOptions = this._options.babelOptions;
+
+        var babelOptions = _.merge(
+                this._options.babelOptions || {},
```
Comment (s-panferov) [inline techs/js-babel.js]: Please use proper indentation here

**Comment by tyv** [comment]
@s-panferov  updated

**Comment by s-panferov** [comment]
@tyv thanks! Merged with a few minor fixes.

**Comment by s-panferov** [comment]
Landed as v0.3.2

**Pull request merged**